### ISSUE TITLE
Fix log level

### ISSUE THREAD

**Issue by koppor** [subhramit/jabref#23]
Logging is great. See for details at https://devdocs.jabref.org/code-howtos/logging.html

However, please do NOT log everything with INFO if it should be DEBUG. Please adapt `tinylog.properties`

Following has to be DEBUG

```
INFO: Attempting to get bibliography range
INFO: Bibliography range found: true
  <div class="csl-entry">
    <div class="csl-left-margin">[1]</div><div class="csl-right-inline">R. Corti, A. J. Flammer, N. K. Hollenberg, and T. F. L&uuml;scher, &ldquo;Cocoa and Cardiovascular Health,&rdquo; <span style="font-style: italic">Circulation</span>, vol. 119, no. 10, pp. 1433&ndash;1441, Mar. 2009, doi: 10.1161/circulationaha.108.827022.</div>
  </div>
INFO: Bibliography inserted using CSLCitationOOAdapter
INFO: CSL bibliography section population completed
INFO: Finished rebuilding CSL bibliography
```

**Comment by subhramit** [comment]
Fixed in https://github.com/JabRef/jabref/pull/11577/commits/3e37e665abbac7cb2a24e3b2b9910529ff3f3295
Also used slf4j logger instead of java.util or tinylog

**Comment by koppor** [comment]
> Fixed in [JabRef@3e37e66](https://github.com/JabRef/jabref/commit/3e37e665abbac7cb2a24e3b2b9910529ff3f3295) Also used slf4j logger instead of java.util or tinylog

We should have an architecture test for that. So that new contributors don't do the same detour :)

**Comment by subhramit** [comment]
> > Fixed in [JabRef@3e37e66](https://github.com/JabRef/jabref/commit/3e37e665abbac7cb2a24e3b2b9910529ff3f3295) Also used slf4j logger instead of java.util or tinylog
> 
> We should have an architecture test for that. So that new contributors don't do the same detour :)

+1
